akka-docs/rst/java/remoting-artery.rst

```diff
@@ -87,6 +87,9 @@ As you can see in the example above there are four things you need to add to get
 The example above only illustrates the bare minimum of properties you have to add to enable remoting.
 All settings are described in :ref:`remote-configuration-artery-java`.
 
+.. warning::
+  Aeron requires 64bit JVM for reliable work. 
```

Comment:
same thing in `scala/remoting-artery.rst`

not sure that this deserves `warning` (red box). I think `note` is enough. 64 bit is default in most environments